### ISSUE TITLE
Move the `cpr` package into DAQ release

### ISSUE THREAD

**Issue by dingp** [DUNE-DAQ/influxopmon#1]
I just found the `develop` head of this package now uses the `cpr` package. 

`cpr` has not been included in DAQ release yet. It still lives under `products_dev` on cvmfs (or `dunedaq-development` cvmfs repo). 

The latest usage of `cpr` in `develop` will break the nightly build unless we move `cpr` into the release. @glehmannmiotto Shall I make the move?

**Comment by dingp** [comment]
Additionally, the package now requires `libcurl`. So `libcurl-devel` will be in the essential system packages list.

**Comment by dingp** [comment]
We've agreed that the `develop` release should be able to use UPS products in `products_dev` or `dunedaq-development.opensciencegrid.org`, which is where `cpr` currently lives.

I've updated the develop release configuration to make use of that, and updated our docker image to include products from `products_dev` in the embedded cvmfs mirror.

Closing the issue for the moment. Will reopen when ready to move `cpr` to `products`.
